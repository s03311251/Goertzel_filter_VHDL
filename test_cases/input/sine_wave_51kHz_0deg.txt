2000
2A14
3321
3A3C
3EAB
3FFB
3E09
3908
317B
2826
1DFD
1408
0B4B
04AA
00D1
0024
02B6
0844
103B
19CE
2402
2DCE
3632
3C53
3F92
3F9A
3C6B
3657
2DFC
2435
1A01
1068
0866
02CB
0029
00C5
048F
0B24
13D9
1DCA
27F5
3150
38E8
3DF7
3FF9
3EBA
3A59
334A
2A45
2033
161C
0D07
05E0
0163
0003
01E4
06D7
0E59
17A7
21CE
2BC7
348D
3B3A
3F23
3FDF
3D5D
37DE
2FF0
2663
1C30
1260
09F3
03C4
0076
005D
037D
0984
11D5
1B97
25CC
2F6A
3776
3D1F
3FD0
3F45
3B8A
3502
2C56
2269
183C
0EDA
0737
021A
0008
0137
0588
0C8C
158A
1F99
29B2
