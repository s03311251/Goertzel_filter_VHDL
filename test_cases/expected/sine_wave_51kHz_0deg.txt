FFED44C0
FFEC4C40
